### ISSUE TITLE
Enhance a11y for `RangeDatePicker`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1253]
path: stories/components/forms/range-date-picker.stories.tsx

### Description
0 accessibility violation was found in the `RangeDatePicker`.

### Problem Statement/Justification
A11y is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all jobs, if there are too many jobs to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
